packages/management/tkg/bundle/config/values.yaml

```diff
@@ -16,6 +16,33 @@ frameworkPackage:
   tkrServicePackageValues:
     namespace: tkg-system
     createNamespace: false
+  addonsManagerPackageValues:
+    tanzuAddonsManager:
+      namespace: tkg-system
+      createNamespace: false
+      deployment:
+        hostNetwork: true
+        priorityClassName: system-cluster-critical
+        nodeSelector:
+          node-role.kubernetes.io/master: ""
+        tolerations:
+        - effect: NoSchedule
+          key: node-role.kubernetes.io/master
+          operator: Exists
+        - key: CriticalAddonsOnly
+          operator: Exists
+        - effect: NoExecute
+          key: node.alpha.kubernetes.io/notReady
+          operator: Exists
+        - effect: NoExecute
+          key: node.alpha.kubernetes.io/unreachable
+          operator: Exists
+        - effect: NoSchedule
+          key: kubeadmNode
+          operator: Equal
+          value: master
+      featureGates:
+        clusterBootstrapController: true
```

Comment:
Should be false by default. CLI/supervisor install will need to plumb this to true based on a higher level feature flag.

Reply:
Updated.